resources/testdriver.js

```diff
@@ -1023,7 +1023,15 @@
          */
         get_virtual_sensor_information: function(sensor_type, context=null) {
             return window.test_driver_internal.get_virtual_sensor_information(sensor_type, context);
-        }
+        },
```

Comment:
Notice how the last item didn't have a comma but your additions do.

Reply:
Comma removed in https://github.com/JuhaVainio/WPT/pull/4/commits/0ff7d73903eda5ad4a795743a15a623ab5f9ca4b